### PR TITLE
[quant][tensorrt] Add tensorrt backend config

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #64623

Summary:
The config api will change, but we'll add configs gradually for TensorRT to unblock experimentation

Test Plan:
python torch/fx/experimental/fx2trt/example/unittests.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30800474](https://our.internmc.facebook.com/intern/diff/D30800474)

### CODE REVIEW

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30800474).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30800474).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30800474).

File: torch/quantization/fx/quantization_patterns.py [hunk 1]
```diff
@@ -1690,3 +1689,309 @@ def convert(self,
         setattr(modules[parent_name], name, quantized_standalone_module)
         modules[str(node.target)] = quantized_standalone_module
         return quantized_graph.node_copy(node, load_arg(quantized=input_quantized_idxs))
+
+
+class ConvReLUQuantizeHandlerNew(QuantizeHandler):
```
Comment (vkuzo) [inline torch/quantization/fx/quantization_patterns.py:1694]: nit: would it be possible to put these in a separate file while the structure is temporary, so it's easier for other people to understand the rest of `quantization_patterns.py`?

**Review by vkuzo — APPROVED** [review]

File: torch/quantization/fx/quantization_patterns.py (hunk 1 above)
Comment (jerryzh168) [reply torch/quantization/fx/quantization_patterns.py:1694]: sure, can I land first and follow up later, planning to do migration asap after this PR, looks like it takes time to import things for these handlers in the new file.

**Pull request closed without merging**